iris_grib/tests/integration/save_rules/test_grib_save.py

```diff
@@ -0,0 +1,326 @@
+        cube.coord(dimensions=[0]).coord_system = None
+        saved_grib = iris.util.create_temp_filename(suffix=".grib2")
+        self.assertRaises(
+            iris.exceptions.TranslationError, iris.save, cube, saved_grib
+        )
+        os.remove(saved_grib)
+
+    def test_forecast_period(self):
+        # unhandled unit
+        cube = self._load_basic()
+        cube.coord("forecast_period").units = cf_units.Unit("years")
+        saved_grib = iris.util.create_temp_filename(suffix=".grib2")
+        self.assertRaises(
+            iris.exceptions.TranslationError, iris.save, cube, saved_grib
+        )
+        os.remove(saved_grib)
+
+    def test_unhandled_vertical(self):
+        # unhandled level type
+        cube = self._load_basic()
+        # Adjust the 'pressure' coord to make it into an "unrecognised Z coord"
+        p_coord = cube.coord("pressure")
+        p_coord.rename("not the messiah")
+        p_coord.units = "K"
+        p_coord.attributes["positive"] = "up"
+        saved_grib = iris.util.create_temp_filename(suffix=".grib2")
+        with self.assertRaises(iris.exceptions.TranslationError):
+            iris.save(cube, saved_grib)
+        os.remove(saved_grib)
+
+    def test_scalar_int32_pressure(self):
+        # Make sure we can save a scalar int32 coordinate with unit conversion.
+        cube = self._load_basic()
+        cube.coord("pressure").points = np.array([200], dtype=np.int32)
+        cube.coord("pressure").units = "hPa"
+        with self.temp_filename(".grib2") as testfile:
+            iris.save(cube, testfile)
+
+    def test_bounded_level(self):
+        cube = iris.load_cube(
+            tests.get_data_path(("GRIB", "uk_t", "uk_t.grib2"))
+        )
+        # Changing pressure to altitude due to grib api bug:
+        # https://github.com/SciTools/iris/pull/715#discussion_r5901538
+        cube.remove_coord("pressure")
+        cube.add_aux_coord(
+            iris.coords.AuxCoord(
+                1030.0,
+                long_name="altitude",
+                units="m",
+                bounds=np.array([111.0, 1949.0]),
+            )
+        )
+        with self.temp_filename(".grib2") as testfile:
+            iris.save(cube, testfile)
+            with open(testfile, "rb") as saved_file:
+                g = gribapi.grib_new_from_file(saved_file)
+                self.assertEqual(
+                    gribapi.grib_get_double(
+                        g, "scaledValueOfFirstFixedSurface"
+                    ),
+                    111.0,
+                )
+                self.assertEqual(
+                    gribapi.grib_get_double(
+                        g, "scaledValueOfSecondFixedSurface"
+                    ),
+                    1949.0,
+                )
+
+
+class TestHandmade(tests.IrisGribTest):
+    def _lat_lon_cube_no_time(self):
+        """
+        Returns a cube with a latitude and longitude suitable for testing
+        saving to PP/NetCDF etc.
```

Comment:
I suppose this no longer makes sense in the context of testing for GRIB. 
I'll change this docstring